extension/src/background/index.ts

```diff
@@ -54,8 +54,8 @@ export const initInstalledListener = () => {
       default:
     }
   });
-  browser?.runtime?.onInstalled.addListener(migrateFriendBotUrlNetworkDetails);
   browser?.runtime?.onInstalled.addListener(normalizeMigratedData);
+  browser?.runtime?.onInstalled.addListener(migrateFriendBotUrlNetworkDetails);
```

Comment:
unrelated, but we need to do the friendbot migration after normalize or else the browser.storage data that needs to be updated may not exist yet